src/randoop/main/ClassTypeLoader.java

```diff
@@ -0,0 +1,134 @@
+package randoop.main;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import randoop.types.TypeNames;
+import randoop.util.Files;
+
+import plume.EntryReader;
+
+/**
+ * ClassTypeLoader provides methods that allow a list of class names to be loaded
+ * as {@link Class<?>} objects.
+ * 
+ * (Extracted from class previously known as randoop.util.Reflection.)
+ * 
+ * @author bjkeller
+ *
+ */
+public class ClassTypeLoader {
+
+  /**
+   * loadClassesFromList returns a list of {@link Class} objects given a list of class names.
+   * In the case that the typename recognition throws an exception, an error is only thrown if
+   * <code>noerr</code> is false.
+   * 
+   * @param classNames list of strings for classnames.
```

Comment:
I would just say "list of fully-qualified class names".  It will be obvious they are strings.  If you want to say that, put it later rather than first because it's not the key information.